envoy/config/rbac/v2alpha/local_rbac.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+package envoy.config.rbac.v2alpha;
+
+import "envoy/config/rbac/v2alpha/rbac.proto";
+
+// LocalRBAC specifies RBAC policies that apply to the local RBAC engine
```

Comment:
Do we really need to know whether the RBAC is local or remote? Who should be reader of this info?

Reply:
"local" means the RBAC filter spec seen by the proxy. I explained it in README.md.